environment/src/main/resources/schema/app/20200930134106_CB-8867-Env_service_should_create_private_DNS_zone_privatelink.postgres.database.azure.com.sql

```diff
@@ -0,0 +1,23 @@
+-- // CB-8867-Env service should create private DNS zone privatelink.postgres.database.azure.com
+-- Migration SQL that makes the change goes here.
+CREATE SEQUENCE IF NOT EXISTS resource_id_seq START WITH 1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
+
+CREATE TABLE IF NOT EXISTS resource
+(
+   id                   bigserial NOT NULL ,
+   resource_environment     bigserial NOT NULL CONSTRAINT fk_resource_resource_environment
```

Comment:
as mentioned above I would name this `environment_id` and also I would add index to this field

Reply:
Right now, we are not using env_id in any of our selects, I would put the index when it is actually useful.

Reply:
I found one method in the repository which has it. Also I'm afraid we will miss it when someone adds one, and notice it only when it's causing performance issues

Reply:
It is not called yet, but I've created the index.